src/tir/schedule/concrete_schedule.cc

```diff
@@ -452,6 +452,9 @@ Array<LoopRV> ConcreteScheduleNode::Split(const LoopRV& loop_rv,
       if (is_const_int(factor) && !is_positive_const(factor)) {
         throw NonPositiveFactorError(state_->mod, factor.as<IntImmNode>()->value, i);
       }
+      if (factor.dtype().bits() > loop->extent.dtype().bits()) {
+        factor = tir::Cast(loop->extent.dtype(), factor);
```

Comment:
use lower cased cast

Reply:
Do you mean `tvm::cast` not `tir::cast` right? There seems to be no lowercased cast in tir namespace.

Reply:
right